test/package_api_storage_test.dart

```diff
@@ -7,8 +7,9 @@ import 'package:test/test.dart';
 void main() {
   group('PackageApi storage tests', () {
     test('Storage v3 gets exported correctly', () {
-      final typedJson =
-          PackageApiStorage.packageApitoStorageJson(testPackage2Api);
+      final typedJson = PackageApiStorage.packageApitoStorageJson(
+        testPackage2Api,
```

Comment:
NIT: Not sure about the naming convention here, but after removing other tested objects we could probably remove the number, or start numbering from 1.
```suggestion
        testPackageApi,
```